src/Config/AuthToken.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Config;
+
+/**
+ * Authenticator Configuration for Token Auth and HMAC Auth
+ */
+class AuthToken
```

Comment:
You add a new Config file. so update the installation instruction:
https://codeigniter4.github.io/shield/install/#initial-setup